homework1-solution/pycharm-project/read_and_insert_dataset.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+import dataset
+import datetime
+
+orders = pd.read_csv("orders.csv")
+employees = pd.read_csv('employees.csv')
+products = pd.read_csv('products.csv').drop_duplicates()
+
+empl = employees[['employee_number', 'last_name', 'first_name', 'reports_to', 'job_title', 'office_code']].drop_duplicates()
+#print([products.isna().any()])
```

Comment:
Cleanup comments! No commented out code in commited code. That's what git is for, you can keep old code in old commits, and return as needed.